### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vdubr**
# Version

Published prerelease version: `v1.24.7-dev.7`

<details>
  <summary>Changelog</summary>

  #### ⚠️ Pushed to `dev`
  
  - Implement hovered pixel color in WMS layer ([@vdubr](https://github.com/vdubr))
  - Typo ([@vdubr](https://github.com/vdubr))
  - Add todo to replace readPixelsToArray in WMS layer ([@vdubr](https://github.com/vdubr))
  - Handle feature IDs in mvt and vector layer ([@vdubr](https://github.com/vdubr))
  - Update dependencies ([@vdubr](https://github.com/vdubr))
  - Temporary fix lumagl/webgl ([@vdubr](https://github.com/vdubr))
  - Update cogTerrain layer props ([@vdubr](https://github.com/vdubr))
  - Update dev package ([@vdubr](https://github.com/vdubr))
  - Test deckgl-geotiff dev package ([@vdubr](https://github.com/vdubr))
  
  #### Authors: 1
  
  - Vojtěch Dubrovský ([@vdubr](https://github.com/vdubr))
</details>

### CODE REVIEW

File: src/DeckGlMap/index.js [hunk 1]
```diff
@@ -74,15 +73,16 @@ const DeckGlMap = ({
...
+						// TODO replace readPixelsToArray
+						// const image =
+						// 	item?.tile?.layers?.[0]?.props?.tile?.layers?.[0]?.props?.image;
+						// item.pixelColor = readPixelsToArray(image, {
+						// 	sourceX: event?.bitmap?.pixel?.[0],
+						// 	sourceY: event?.bitmap?.pixel?.[1],
+						// 	sourceWidth: 1,
+						// 	sourceHeight: 1,
+						// });
+						// rasterHoveredItems.push(item);
```
Comment (vlach1989) [inline src/DeckGlMap/index.js:85]: why is this part commented?

File: src/DeckGlMap/layers/utils/geojson-binary.js [hunk 2]
```diff
@@ -4,21 +4,19 @@
 // https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/utils.ts
 
 /**
- * Return the feature for an accessor
+ * Return the feature for an accesor
  */
-export function binaryToFeatureForAccessor(data, index, indices) {
+export function binaryToFeatureForAccesor(data, index) {
```
Comment (vlach1989) [inline src/DeckGlMap/layers/utils/geojson-binary.js:9]: typo in Accessor

**Review by vlach1989 — APPROVED** [review]

File: src/DeckGlMap/layers/utils/geojson-binary.js (hunk 2 above)
Comment (vdubr) [reply src/DeckGlMap/layers/utils/geojson-binary.js]: fixed

File: src/DeckGlMap/index.js (hunk 1 above)
Comment (vdubr) [reply src/DeckGlMap/index.js]: Uncommented and implemented

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**